openapi/SwarmCommon.yaml

```diff
@@ -634,7 +634,7 @@ components:
     GetStakeResponse:
       type: object
       properties:
-        stakedAmount:
```

Comment:
API spec version should be updated to reflect the breaking change.

Reply:
The API version was already updated for Bee 2.2. As I know it should be updated(only if there are any API changes) once per new Bee release.